File: WebService/src/main/java/com/dlgdev/pokemon/webservice/dagger/FrontControllerComponent.java
package com.dlgdev.pokemon.webservice.dagger;

import com.dlgdev.pokemon.webservice.FrontController;

import dagger.Component;

@Component(modules = {LoggerModule.class, DatabaseModule.class})
public interface FrontControllerComponent {

	void inject(FrontController frontController);
}

File: WebService/src/main/kotlin/com/dlgdev/pokemon/webservice/dagger/FrontControllerComponent.kt
package com.dlgdev.pokemon.webservice.dagger

import com.dlgdev.pokemon.webservice.controllers.FrontController

import dagger.Component

@Component(modules = arrayOf(LoggerModule::class, MariaDbModule::class, GsonModule::class))
interface FrontControllerComponent {
    fun inject(frontController: FrontController)
}
